Taskwarrior Pomodoro/AppDelegate.swift

```diff
@@ -367,11 +472,12 @@ class AppDelegate: NSObject, NSApplicationDelegate, NSMenuDelegate {
     func startTaskById(taskId: String) {
         activeTaskId = taskId
         
-        let task = NSTask()
-        task.launchPath = taskPath
-        task.arguments = [taskId, "start"]
-        task.launch()
-        task.waitUntilExit()
+        taskCommand([taskId, "start"])
+        activeTaskPomodorosLogUUID = currentPomodorosLogUUID
```

Comment:
Oh!  I totally didn't catch that `currentPomodorosLogUUID` would sometimes be `nil`, and that it was set via `menuWillOpen`.  Cool; sounds good to me, and sorry for misunderstanding!